arch/arm64/boot/dts/qcom/sm7150-samsung-a715f.dts

```diff
@@ -0,0 +1,788 @@
+// SPDX-License-Identifier: BSD-3-Clause
```

Comment:
Look at how commits are formatted with other devices (preferably those that are already in upstream). I mean commit name and message